nixos/modules/services/misc/gitea.nix

```diff
@@ -454,6 +471,41 @@ in
       ];
     };
 
+    services.nginx = lib.mkIf cfg.configureNginx {
+      enable = true;
+      virtualHosts."${cfg.domain}" = {
+        root = "/var/empty";
+
+        locations."/" = {
+          proxyPass = (if cfg.enableUnixSocket then "http://unix:/run/gitea/gitea.sock" else "http://127.0.0.1:${toString(cfg.httpPort)}");
```

Comment:
enableUnixSocket should be the default

Reply:
```suggestion
          proxyPass = (if cfg.enableUnixSocket then "http://unix:/run/gitea/gitea.sock" else "http://127.0.0.1:${toString cfg.httpPort}");
```

Reply:
> enableUnixSocket should be the default

If set `services.gitea.enableUnixSocket = true` to default, then gitea won't work without nginx.

Reply:
Can we make it the default when nginx is used? That should work.

Reply:
This variant doesn't want to work:
```
       at /nix/store/p5gbiqgy4482pparylp4z0r0qw4irgs5-nixos/modules/services/misc/gitea.nix:253:9:

          252|         type = types.bool;
          253|         default = if cfg.configureNginx then true else false;
             |         ^
          254|         description = lib.mdDoc "Configure Gitea to listen on a unix socket instead of the default TCP port.";

       error: attribute 'services' missing

```

Reply:
Strange, the test works, but the system update causes an error.

Reply:
Fixed!